macOS/LocalPackages/FeatureFlags/Sources/FeatureFlags/FeatureFlag.swift

```diff
@@ -193,6 +193,9 @@ public enum FeatureFlag: String, CaseIterable {
     /// https://app.asana.com/1/137249556945/project/1211150618152277/task/1208865987567163?focus=true
     case themes
 
+    /// https://app.asana.com/1/137249556945/project/1204006570077678/task/1211258257937392?focus=true
+    case appStoreCheckForUpdatesFlow
```

Comment:
Can you add this flag to the "Updates" Category so it shows up in Debug Menu > Feature Flag Overrides > Updates?

Reply:
Sure!